Sources/UIKit+Diff.swift

```diff
@@ -15,21 +15,10 @@ public extension UITableView {
                       deleteRowAnimation: UITableViewRowAnimation = .automatic)
     {
         beginUpdates()
-        var insertionIndexPaths: [IndexPath] = []
-        var deletionIndexPaths: [IndexPath] = []
-        for step in diff {
-            switch step {
-            case .insert(let index):
-                insertionIndexPaths.append(IndexPath(row: index, section: section))
-            case .delete(let index):
-                deletionIndexPaths.append(IndexPath(row: index, section: section))
-            case .move(let from, let to):
-                moveRow(at: IndexPath(row: from, section: section),
-                                   to: IndexPath(row: to, section: section))
-            }
-        }
-        insertRows(at: insertionIndexPaths, with: insertRowAnimation)
-        deleteRows(at: deletionIndexPaths, with: deleteRowAnimation)
+        applyWithoutBatch(diff: diff,
```

Comment:
hmm I'm not sure about the name `applyWithoutBatch` but I don't think I like `applyWithinBatch` either. If I wasn't looking at the code and read `applyWithinBatch` i would assume that the method was going to apply the diff within a batch so I wouldn't put it in a batch. /shrug maybe thats just how I am reading it though?